src/Goldilocks/Program.cs

```diff
@@ -1,43 +1,29 @@
 using System.Text.Json.Serialization;
+using Goldilocks;
 
 var builder = WebApplication.CreateBuilder(args);
+builder.Logging.ClearProviders();
```

Comment:
I realize that by clearing all the providers (including the console) then we can't detect when the application is ready: `Application started.`

the mitigation is to add this before `app.Run()`:

```
app.Lifetime.ApplicationStarted.Register(() => Console.WriteLine("Application started. Press Ctrl+C to shut down."));
```

Reply:
Oh good point I forgot how benchmarks figures out when to start injecting load ;)